staging/kos/pkg/networkfabric/ovs/ovs.go

```diff
@@ -966,15 +965,23 @@ func (f *ovsFabric) newGetFlowsCmd() *exec.Cmd {
 	return exec.Command("ovs-ofctl", "dump-flows", f.bridge)
 }
 
-func (f *ovsFabric) newListOFportsAndIfcNamesCmd() *exec.Cmd {
+func (f *ovsFabric) newListIfcsNamesCmd() *exec.Cmd {
 	return exec.Command("ovs-vsctl",
-		"-f",
-		"table",
-		"--no-heading",
-		"--",
-		"--columns=ofport,name",
-		"list",
-		"Interface")
+		"list-ports",
+		f.bridge)
+}
+
+// newGetIfcOFPortsCmd returns a command that lists all the OpenFlow port
+// numbers associated with the interfaces in `ifcs`.
+// For performance reasons, the command runs within a single OvS transaction (a
+// transaction is the only way to have a single OvS command for all interfaces
+// rather than one separate command per interface).
+func (f *ovsFabric) newGetIfcsOFPortsCmd(ifcs []string) *exec.Cmd {
+	cmdArgs := []string{"get", "interface"}
+	ifcsJoinedForCmd := strings.Join(ifcs, " ofport -- get interface ")
```

Comment:
Consider the "--" between "ofport" and "get interface".
It is "ovs-vsctl" syntax for executing multiple commands (separated by "--") within a single transaction. 
Within a single transaction also means within a single command which also means a single fork + exec.
This approach is more scalable than the more intuitive approach of retrieving the OpenFlow port number for eac interface separately (1 fork + exec for every interface); see [here](https://docs.google.com/spreadsheets/d/1d5MesCutm8pINd3YJtHkhj6zwSPoTkkpqALTHXmy6IM/edit?usp=sharing) for a sloppy perf comparison.
The drawback is that we're conflating logically independent commands in a single transaction, if there are many of them the likelihood of at least one of them failing increases and if that happens the whole command fails. But OvS has a reputation for being very reliable, so IMO it is worth it.